lib/api/funnel.js

```diff
@@ -25,16 +25,7 @@ const Bluebird = require('bluebird');
 const Deque = require('denque');
 
 const { KuzzleError } = require('../kerror/errors');
-const AdminController = require('./controller/admin');
-const AuthController = require('./controller/auth');
-const BulkController = require('./controller/bulk');
-const CollectionController = require('./controller/collection');
-const DocumentController = require('./controller/document');
-const IndexController = require('./controller/index');
-const MemoryStorageController = require('./controller/memoryStorage');
-const RealtimeController = require('./controller/realtime');
-const SecurityController = require('./controller/security');
-const ServerController = require('./controller/server');
+const nativeControllers = require('./controllers');
```

Comment:
I would rather keep importing the original class name so it's easier to identify them later
```suggestion
const {
  AdminController,
  AuthController,
  BulkController,
  CollectionController,
  DocumentController,
  IndexController,
  MemoryStorageController,
  RealtimeController,
  SecurityController,
  ServerController,
} = require('./controllers');
```

Reply:
This was part of my thinking process also, BUT, the idea of importing the 'whole' controllers module instead of one by one, was so when we update our controller list, there would be only one place to do so.